tsc/meetings/2023-07-25.md

```diff
@@ -0,0 +1,78 @@
+Minutes from OpenVDB TSC meeting, July 25th, 2023
+
+Attendees: *Jeff* L., *Andre* P, *Dan* B., *Rich* J., *Nick* A.
```

Comment:
wait, I  was also attending this meeting!

Reply:
Woops, guess I'm a bit rusty